### PR TITLE
Fix use of UCTS trigger in interleaved pedestals events

### PR DESCRIPTION
**Pull request by FrancaCassol**
Remove the check of UCTS trigger in interleaved pedestal events, because often wrong.

### CODE REVIEW

File: lstchain/calib/camera/calibration_calculator.py [hunk 1]
```diff
@@ -206,11 +206,8 @@ def process_interleaved(self, event):
         new_ped = False
         new_ff = False
 
-        # if pedestal event (check also the ucts trigger for the moment)
-        if (
-                LSTEventType.is_pedestal(event.r1.tel[self.tel_id].trigger_type) or
-                LSTEventType.is_pedestal(event.lst.tel[self.tel_id].evt.ucts_trigger_type)
```
Comment (MaxNoe) [inline lstchain/calib/camera/calibration_calculator.py:212]: To understand this: the ucts_trigger_type wrongly identifies an event as a pedestal although it is not?

How does that happen?

**Comment by DirkHoffmann** [comment]
> Remove the check of UCTS trigger

Is this because the TTYPE is wrong, or is the TIB information correct in these casees?

File: lstchain/calib/camera/calibration_calculator.py (hunk 1 above)
Comment (rlopezcoto) [reply lstchain/calib/camera/calibration_calculator.py:212]: there is a hardware problem that is being worked out. Until then, we cannot trust on this flag

Comment (MaxNoe) [reply lstchain/calib/camera/calibration_calculator.py:212]: @DirkHoffman seems to disagree that this is a hardware problem

Comment (rlopezcoto) [reply lstchain/calib/camera/calibration_calculator.py:212]: There have been several problems of UCTS/TIB bit disappearance at EVB and/or event class misidentification and there are many open discussion in camera/analysis calls.
The bottomline is that we cannot trust this flag for the time being.

**Comment by FrancaCassol** [comment]
> > Remove the check of UCTS trigger
> 
> Is this because the TTYPE is wrong, or is the TIB information correct in these casees?

For the cases I could see the TIB information is either correct or absent.
But, we will have a more global  view of that after the next data processing.

**Pull request merged**